nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/action/computation/AbstractTransientBlobComputation.java

```diff
@@ -47,6 +47,7 @@ public void init(ComputationContext context) {
         super.init(context);
         try {
             temp = Files.createTempDirectory(metadata().name());
+            temp.toFile().deleteOnExit();
```

Comment:
It is probably better to use `Framework#trackFile`.

Reply:
thought about it but then what to use as ref? `context`? don't think the object will be alive needed time. 
The main complain was that temp folder wasn't cleaned up at server startup

Reply:
We use the file itself as marker.

Reply:
ok, thx